modules/using-cluster-compare-telco-ref.adoc

```diff
@@ -0,0 +1,304 @@
+    - required/networking/sriov/sriovNetworkNodePolicy.yaml
+    - required/networking/sriov/SriovOperatorConfig.yaml
+    - required/networking/sriov/SriovSubscription.yaml
+    - required/networking/sriov/SriovSubscriptionNS.yaml
+    - required/networking/sriov/SriovSubscriptionOperGroup.yaml
+required-other:
+  scheduling:
+    Missing CRs:
+    - required/other/catalog-source.yaml
+    - required/other/icsp.yaml
+required-performance:
+  performance:
+    Missing CRs:
+    - required/performance/PerformanceProfile.yaml
+required-scheduling:
+  scheduling:
+    Missing CRs:
+    - required/scheduling/nrop.yaml
+    - required/scheduling/NROPSubscription.yaml
+    - required/scheduling/NROPSubscriptionNS.yaml
+    - required/scheduling/NROPSubscriptionOperGroup.yaml
+    - required/scheduling/sched.yaml
+required-storage:
+  storage-odf:
+    Missing CRs:
+    - required/storage/odf-external/01-rook-ceph-external-cluster-details.secret.yaml
+    - required/storage/odf-external/02-ocs-external-storagecluster.yaml
+    - required/storage/odf-external/odfNS.yaml
+    - required/storage/odf-external/odfOperGroup.yaml
+    - required/storage/odf-external/odfSubscription.yaml
+No CRs are unmatched to reference CRs <8>
+Metadata Hash: fe41066bac56517be02053d436c815661c9fa35eec5922af25a1be359818f297 <9>
+No patched CRs <10>
+----
+<1> The CR under comparison. The plugin displays each CR with a difference from the corresponding template.
+<2> The template matching with the CR for comparison.
+<3> The output in Linux diff format shows the difference between the template and the cluster CR.
+<4> After the plugin reports the line diffs for each CR, the summary of differences are reported.
+<5> The number of CRs in the comparison with differences from the corresponding templates.
+<6> The number of CRs represented in the reference configuration, but missing from the live cluster.
+<7> The list of CRs represented in the reference configuration, but missing from the live cluster.
+<8> The CRs that did not match to a corresponding template in the reference configuration.
+<9> The metadata hash identifies the reference configuration.
+<10> The list of patched CRs.
+endif::cluster-compare-core,cluster-compare[]
+ifdef::cluster-compare-ran[]
+. Extract the content from the `ztp-site-generate-rhel8` container image by running the following commands::
++
+[source,terminal,subs="attributes+"]
+----
+$ podman pull registry.redhat.io/openshift4/ztp-site-generate-rhel8:v{product-version}
+----
++
+[source,terminal]
+----
+$ mkdir -p ./out
+----
++
+[source,terminal,subs="attributes+"]
+----
+$ podman run --log-driver=none --rm registry.redhat.io/openshift4/ztp-site-generate-rhel8:v{product-version} extract /home/ztp --tar | tar x -C ./out
+----
+
+. Compare the configuration for your cluster to the reference configuration by running the following command:
++
+[source,terminal]
+----
+$ oc cluster-compare -r out/path-in-RAN-container/metadata.yaml
```

Comment:
@lack - Is there an upstream image registry address I can use instead of `registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.18`? This 4.18 downstream tag isn't published yet.

I want to doc the directory structure for RAN side and add the example output for the RAN flow. Thanks for your help!

Reply:
Sure!  The easiest one I know of that's closest to what the downstream will be is the brew build:
- `registry-proxy.engineering.redhat.com/rh-osbs/openshift4-ztp-site-generate:v4.18.0`

Reply:
I updated the RDS docs with the RAN example.